src/services/avert-services.ts

```diff
@@ -0,0 +1,33 @@
+import { AvertRecord } from "@/schema/avert";
+import { AvertModel } from "@/database/avert-model";
+import { Location, PowerPlantClass } from "@/schema/egrid";
+import { Error as CustomError } from "@/schema/error";
```

Comment:
Renaming to `CustomError` is good, we might refactor it in the future so it doesn't conflict with the builtin `Error`